src/components/SideBar.vue

```diff
@@ -1,17 +1,27 @@
 <template>
   <div class="col-sm-3  blog-sidebar">
     <div class="github-button-group">
-      <github-button class="github-button" href="https://github.com/apache/incubator-iotdb/fork" data-icon="octicon-repo-forked" data-size="large" data-show-count="true" aria-label="Fork apache/incubator-iotdb on GitHub">Fork</github-button>
-      <github-button class="github-button" href="https://github.com/apache/incubator-iotdb" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star apache/incubator-iotdb on GitHub">Star</github-button>
+      <github-button class="github-button" href="https://github.com/apache/incubator-iotdb/fork"
```

Comment:
Is this a necessary change? Is only a code format improvement? , Just suggest that we can do this change in another PR. :) , but  I am fine with this change if you really want do that.

Reply:
Ah... it's just an auto format from my IDE🙈...

Reply:
I see, It would be better if we have common code style for all of the IoTDB contributors. :)